src/style.css

```diff
@@ -1,3 +1,65 @@
 body {
   margin: 0;
+  padding: 0;
+  --main-color: #00acdc;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.header__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
```

Comment:
<img width="640" alt="Снимок экрана 2023-09-05 в 16 25 09" src="https://github.com/mate-academy/layout_moyo-header/assets/75898310/31dfd476-ae8d-4b80-ad69-958416fdd2aa">
You can remove `width` and `height` because default size of image is same